### ISSUE TITLE
Progress bar

### ISSUE THREAD

**Issue by hadley** [rstudio/rstudioapi#21]
It would be useful if we could access an RStudio progress bar with the same API as `txtProgressBar`

**Comment by saurfang** [comment]
Just want to say that this would be super useful for R notebook such that interactive use cases can display progress bar without being rendered in the notebook HTML.

**Comment by kevinushey** [comment]
What kind of progress bar do we want to render here? Where should it be rendered?

Perhaps a similar treatment for what we do when executing chunks in a notebook?

<img width="272" alt="screen shot 2017-06-05 at 11 50 43 am" src="https://cloud.githubusercontent.com/assets/1976582/26798434/41f49076-49e5-11e7-8c9b-88465ab62a53.png">

**Comment by hadley** [comment]
Yeah that'd be great. Eventually it could render in the same place when you're running in a chunk, but I'm guessing that's too big to consider now.
